code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -206,19 +206,14 @@
 	switch(choice)
 		if("FORCE")
 			var/obj/item/soulstone/C = src
-			if(!iscarbon(target))		//TODO: Add sacrifice stoning for non-organics, just because you have no body doesnt mean you dont have a soul
-				return 0
-			if(contents.len)
-				return 0
-			var/mob/living/carbon/T = target
+			var/mob/living/T = target

```

Comment:
You forgot an istype check here for target prior to parsing

Reply:
None of the other Soul Stone choices use an istype check before parsing target to a type. Also the only two ways of calling the "FORCED" choice is the sacrifice rune here, which won't be called at all unless it's something living, and the unused Cursed Russian Revolver. 

Right now I see adding a check there as redundant, and I would like to know the reasons it's needed if I'm wrong.